src/Microsoft.ComponentDetection.Detectors/pip/PipReportComponentDetector.cs

```diff
@@ -55,6 +62,8 @@ public class PipReportComponentDetector : FileComponentDetector, IExperimentalDe
 
     public override int Version { get; } = 2;
```

Comment:
Since we're changing the logic a bit for Pip and PipReport let's bump detector versions